model/repository/MonitoringRepository.php

```diff
@@ -466,7 +466,13 @@ protected function prepareOrderStmt($order): string
 
```

Comment:
All the functionality within the foreach should be moved to the new method

Reply:
let's try, but i think better say "can" instead "should"